doctr/models/builder.py

```diff
@@ -10,7 +10,7 @@
 from scipy.cluster.hierarchy import fclusterdata
 
 from doctr.io.elements import Block, Document, Line, Page, Word
-from doctr.utils.geometry import rbbox_to_polygon, resolve_enclosing_bbox, resolve_enclosing_rbbox
+from doctr.utils.geometry import rbbox_to_polygon, resolve_enclosing_bbox, resolve_enclosing_rbbox, rotate_boxes
```

Comment:
Watch out, the behaviour of `rotate_boxes` has changed in #488. Make sure you merged "main" into your branch to avoid surprises later on :+1:

Reply:
yes, this function was created in #488